packages/phishing-controller/src/utils.ts

```diff
@@ -223,3 +223,22 @@ export const matchPartsAgainstList = (source: string[], list: string[][]) => {
     return target.every((part, index) => source[index] === part);
   });
 };
+
+/**
+ * Formats a hostname into a URL so we can parse it correctly
+ * and pass full URLs into the PhishingDetector class. Previously
+ * only hostnames were supported, but now only full URLs are 
+ * supported since we want to block IPFS CIDs.
+ * 
+ * @param   hostname - the hostname of the URL.
+ * @returns the href property of a URL object.
+ */
+export const formatHostnameToUrl = (hostname: string): string => {
```

Comment:
It seems that this function is only used in the tests. Do we want the package to export this function? If we only want to share it in tests, we've typically solved this by:

- extracting it to a file in `packages/phishing-controller/tests/` (we may have to create that directory first)
- adding `./tests` to `include` in `packages/phishing-controller/tsconfig.json`

Reply:
[69d9047](https://github.com/MetaMask/core/pull/4465/commits/69d9047d28ebd7606baad4adddb009044863efdf)